pkg/bundle/sync_test.go

```diff
@@ -1313,6 +1330,7 @@ func Test_buildSourceBundle(t *testing.T) {
 		expNotFoundError bool
 		expJKS           bool
 		expPKCS12        bool
+		expArbPasswd     bool // should we use arbitrary password instead of default one?
```

Comment:
```suggestion
		expPassword    *string
```

I wonder if this could simplify the test? If JKS/PKCS12 expected: if `expPassword` is set, use the value, otherwise use default password? That would remove the need for a new constant.